portal/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "oo7-portal"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-std = { version = "1.11", features = [ "attributes" ] }
```

Comment:
We have dropped async_std from the main crate, it would be nice to replace it with whetever the macro ends up calling or just using tokio imoh.

Reply:
Ported to tokio.